### ISSUE TITLE
Research additional datasets

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#166]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Extend current datasets like Hausumringe, Tatsaechliche Nutzung, Lod2, etc. for other parts of germany besides bavaria
* Research available ESPON datasets for natural hazards

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
